[2023-09-13T07:38:45.218+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-08T00:00:00+00:00 [queued]>
[2023-09-13T07:38:45.247+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-08T00:00:00+00:00 [queued]>
[2023-09-13T07:38:45.248+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:45.284+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-08 00:00:00+00:00
[2023-09-13T07:38:45.307+0000] {standard_task_runner.py:57} INFO - Started process 146 to run task
[2023-09-13T07:38:45.323+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-08T00:00:00+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpxiyx_a92']
[2023-09-13T07:38:45.326+0000] {standard_task_runner.py:85} INFO - Job 34: Subtask extract_data
[2023-09-13T07:38:45.446+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-08T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:45.710+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-08T00:00:00+00:00'
[2023-09-13T07:38:46.038+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:46.056+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230908T000000, start_date=20230913T073845, end_date=20230913T073846
[2023-09-13T07:38:46.112+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:46.130+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:20.370+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-08T00:00:00+00:00 [queued]>
[2023-09-15T11:31:20.382+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-08T00:00:00+00:00 [queued]>
[2023-09-15T11:31:20.383+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:20.413+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-08 00:00:00+00:00
[2023-09-15T11:31:20.420+0000] {standard_task_runner.py:57} INFO - Started process 379 to run task
[2023-09-15T11:31:20.434+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-08T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp6vtoy_0k']
[2023-09-15T11:31:20.440+0000] {standard_task_runner.py:85} INFO - Job 30: Subtask extract_data
[2023-09-15T11:31:20.550+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-08T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:20.689+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-08T00:00:00+00:00'
[2023-09-15T11:31:20.977+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:21.001+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230908T000000, start_date=20230915T113120, end_date=20230915T113121
[2023-09-15T11:31:21.042+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:21.059+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
